client/my-sites/people/invite-people/index.jsx

```diff
@@ -387,7 +387,7 @@ class InvitePeople extends React.Component {
 						</FormFieldset>
 
 						<FormButton disabled={ this.isSubmitDisabled() } onClick={ onClickSendInvites }>
-							{ translate( 'Send Invitation', 'Send Invitations', {
+							{ translate( 'Send invitation', 'Send invitations', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.